resources/services/ec2/nat_gateways.go

```diff
@@ -132,7 +132,7 @@ func Ec2NatGateways() *schema.Table {
 				Name:        "aws_ec2_nat_gateway_addresses",
 				Description: "Describes the IP addresses and network interface associated with a NAT gateway.",
 				Resolver:    fetchEc2NatGatewayAddresses,
-				Options:     schema.TableCreationOptions{PrimaryKeys: []string{"nat_gateway_cq_id", "allocation_id", "network_interface_id"}},
```

Comment:
As another option we can drop PK completely on this subtable, but I think it should be fine just without allocation_id. (at least it fetches all our test resources).

Reply:
Do we want to add private_ip to the PK?